api/base.proto

```diff
@@ -175,6 +175,11 @@ message ApiConfigSource {
   ApiType api_type = 1 [(validate.rules).enum.defined_only = true];
   // Multiple cluster names may be provided. If > 1 cluster is defined, clusters
   // will be cycled through if any kind of failure occurs.
+  //
+  // .. note::
+  //
+  //  The cluster with name ``cluster_name`` must be statically defined, or an
+  //  EnvoyException will be thrown.
```

Comment:
`EnvoyException` doesn't have any meaning to a doc reader. (These are user docs). Can you update? Same below.